Domains/4-Application/Connectors/Navisworks/NwdStatic.ecschema.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- ==================================================================================
+|  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+|  * See LICENSE.md in the project root for license terms and full copyright notice.
+======================================================================================= -->
+<ECSchema schemaName="NwdStatic" description="NWD connector static schema" alias="nwds" version="01.00.01" xmlns="http://www.bentley.com/schemas/Bentley.ECXML.3.2">
```

Comment:
Please add an entry for this work-in-progress schema in the `SchemaInventory.json`.